temporalio/worker/_workflow_instance.py

```diff
@@ -158,6 +159,16 @@ def activate(
         """
         raise NotImplementedError
 
+    def get_worker_ident(self) -> Optional[int]:
+        """Return the thread identifier that this workflow is running on.
```

Comment:
This is called `get_worker_ident` but is returning the id of a thread. The name should probably be `get_thread_id` or something

Reply:
Also rather than raising NotImplemented, probably just return `None` here by default.